tracer/src/Datadog.Tracer.Native/probes_tracker.cpp

```diff
@@ -49,11 +49,17 @@
      if (_probeMetadataMap.find(probeId) == _probeMetadataMap.end())
      {
          auto methods = std::set<trace::MethodIdentifier>();
-         _probeMetadataMap[probeId] = std::make_shared<ProbeMetadata>(probeId, std::move(methods), ProbeStatus::RECEIVED);
+         auto indices = std::set<int>();
+         _probeMetadataMap[probeId] = std::make_shared<ProbeMetadata>(probeId, std::move(methods), ProbeStatus::RECEIVED, std::move(indices));
      }
  }
 
- void debugger::ProbesMetadataTracker::AddMethodToProbe(const shared::WSTRING& probeId, const ModuleID moduleId, const mdMethodDef methodId)
+bool debugger::ProbesMetadataTracker::IsProbeExists(const shared::WSTRING& probeId)
```

Comment:
```suggestion
bool debugger::ProbesMetadataTracker::ProbeExists(const shared::WSTRING& probeId)
```

Reply:
Done in 90e0ba24cfd43eab1fca79e8a887db33e9ddf49e.